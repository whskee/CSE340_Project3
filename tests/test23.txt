a, b: int;
c: int;
{
    x = + a * b c;
    y = ! true;
}
