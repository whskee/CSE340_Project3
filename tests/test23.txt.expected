a: int #
b: int #
c: int #
x: int #
y: bool #
